Open Judge System/OJS.Common/GlobalConstants.cs

```diff
@@ -75,6 +76,9 @@ public static class GlobalConstants
         public const string ExecutableFileExtension = ".exe";
         public const string ClassLibraryFileExtension = ".dll";
 
+        // Folder names
+        public const string ExecutionStrategyTemp = "ExecutionStrategyTemp";
```

Comment:
Is this the name of the folder where all the execution strategies will have their working directories? If so, then change it to plural  i.e `Strategies` and add `FolderName` and remove `Temp` since it will not be _temporary_ per say